server/src/test/java/io/spine/server/projection/ProjectionRepositoryShould.java

```diff
@@ -504,63 +307,57 @@ public void convert_null_timestamp_to_default() {
     }
 
     @Test
-    public void do_not_create_record_if_entity_isnt_updated() {
+    public void do_not_create_record_if_entity_is_not_updated() {
         final NoOpTaskNamesRepository repo = new NoOpTaskNamesRepository();
         boundedContext.register(repo);
 
         assertFalse(repo.loadAll().hasNext());
 
-        final Event event = createEvent(PRODUCER_ID, projectCreated());
+        final Event event = createEvent(tenantId(),
+                                        projectCreated(),
+                                        PRODUCER_ID,
+                                        getCurrentTime());
         repo.dispatch(EventEnvelope.of(event));
 
         final Iterator<?> items = repo.loadAll();
         assertFalse(items.hasNext());
     }
 
-    private static ManualCatchupProjectionRepository repoWithManualCatchup() {
-        final ManualCatchupProjectionRepository repo =
-                new ManualCatchupProjectionRepository();
-        return repo;
+    /**
+     * Ensures that {@link ProjectionRepository#getIdSetFunction(Class)} which is used by Beam-based
```

Comment:
Same as above. Ideally this module should know nothing about Beam. Even in test comments.